src/components/Application.tsx

```diff
@@ -3,23 +3,24 @@ import * as React from 'react'
 import { Redirect, Route, RouteComponentProps, Switch } from 'react-router'
 
 import DashboardContainer from '../containers/DashboardContainer'
-import { IApplicationState } from '../reducers/application'
+import PlaygroundContainer from '../containers/PlaygroundContainer'
 import NavigationBar from './NavigationBar'
 import NotFound from './NotFound'
 
 export interface IApplicationProps extends RouteComponentProps<{}> {
-  application: IApplicationState
+  title: string
 }
 
-const Application: React.SFC<IApplicationProps> = ({ application }) => {
+const Application: React.SFC<IApplicationProps> = (props: IApplicationProps) => {
```

Comment:
Adding `IApplicationProps` after `props:` is redundant because we already declare the type as `React.SFC<IApplicationProps>`. TypeScript is able to infer the parameter type. So you can unpack `{ title }` straight away in the parameter.

Reply:
I feel like it would be better to keep a `props` argument, so that adding properties to `IApplicationProps` does not require an additional change in the arguments. What do you think?